specs/_features/ePBS/p2p-interface.md

```diff
@@ -0,0 +1,119 @@
+# ePBS -- Networking
+
+
+This document contains the consensus-layer networking specification for ePBS.
+
+The specification of these changes continues in the same format as the network specifications of previous upgrades, and assumes them as pre-requisite.
+
+## Table of contents
+
+### The gossip domain: gossipsub
+
+Some gossip meshes are upgraded in the fork of ePBS to support upgraded types.
+
+#### Topics and messages
+
+Topics follow the same specification as in prior upgrades.
+
+The `beacon_block` topic is modified to also support block with bid and new topics are added per table below.
+
+The derivation of the `message-id` remains stable.
+
+The new topics along with the type of the `data` field of a gossipsub message are given in this table:
+
+| Name | Message Type |
+| - | - |
+| `builder_bid` | `SignedBuilderBid`|
+| `execution_payload` | `ExecutionPayload`|
+| `execution_attestation` | `ExecutionAttestation` [New in ePBS]
+
+##### Global topics
+
+ePBS introduces new global topics for blob sidecars.
+
+###### `beacon_block`
+
+The *type* of the payload of this topic changes to the (modified) `SignedBeaconBlockWithBid` found in ePBS spec. Specifically, this type changes with the replacement of `SignedBuilderBid` to the inner `BeaconBlockBody`.
+
+In addition to the gossip validations for this topic from prior specifications, the following validations MUST pass before forwarding the `signed_beacon_block` on the network. Alias `block = signed_beacon_block.message`, `signed_bid = block.body.signed_builder_bid`, `bid = signed_bid.message`, `header = bid.header`.
+
+
+New validation:
+
+- _[REJECT]_ The block's execution header timestamp is correct with respect to the slot -- i.e. `header.timestamp == compute_timestamp_at_slot(state, block.slot)`.
+- _[REJECT]_ The bid pubkey is a valid builder in state.
+- _[REJECT]_ The builder has sufficient balances to pay for the bid.
```

Comment:
how would this be validated? won't this require access to execution state?

Reply:
Bid payment is happening at consensus, not execution. @potuz should have a consensus spec soon